mailbox/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -30,6 +30,9 @@
         <class>org.apache.james.mailbox.jpa.mail.model.openjpa.JPAMailboxMessage</class>
         <class>org.apache.james.mailbox.jpa.mail.model.JPAProperty</class>
         <class>org.apache.james.mailbox.jpa.user.model.JPASubscription</class>
+        <class>org.apache.james.domainlist.jpa.model.JPADomain</class>
```

Comment:
Yes, I added new persistence with full model and revert the old files as they are.